### DISCUSSION TITLE
Corrupted file when saving RGBA image saved as TIFF CcittGroup4Fax

### DISCUSSION THREAD

**Discussion by ribeiroi** [SixLabors/ImageSharp#2539] [Q&A]
Hello,
I have a RGBA32 image. It can be saved to PNG with no problem.
When saved to TIFF using CcittGroup4Fax, it generates a corrupted file (can't be opened). If I change compression to LZW, it can be opened but of course file is much bigger.

Is this a bug? I need to save the RGBA32 image as a TIFF CcittGroup4Fax.

Thank you.

**Comment by brianpopow** [comment]
@ribeiroi without an example image, there is not much we can do about it.

**Reply by JimBobSquarePants** [reply]
@ribeiroi Are you able to provide the file?
